examples/groth16/README.md

```diff
@@ -0,0 +1,7 @@
+# Sui Groth16 Example
+This is a simple example to the Sui Groth16 documentation.
```

Comment:
```suggestion
This is an example to the Sui Groth16 documentation.
```
Avoid using the term 'simple' in tech writing. If it's not simple for whatever reason to the user, you're essentially calling them dumb.